core/src/autogluon/core/_setup_utils.py

```diff
@@ -10,7 +10,7 @@
     os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '..', '..')
 )
 
-PYTHON_REQUIRES = '>=3.6, <3.8'
+PYTHON_REQUIRES = '>=3.6, <3.9'
```

Comment:
Unrelated to this PR, but I see a warning printed on importing autogluon.vision that can probably be fixed: http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-955/3/tutorials/image_prediction/beginner.html